yt_dlp/extractor/zaiko.py

```diff
@@ -0,0 +1,55 @@
+import json
+import html
+import re
+
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class ZaikoIE(InfoExtractor):
+    _VALID_URL = r'https://zaiko\.io/event/(?P<id>\d+)/stream(?:/\d+)+'
```

Comment:
Match `http` and `www.` too since they work/redirect in browser
```suggestion
    _VALID_URL = r'https?://(?:www\.)?zaiko\.io/event/(?P<id>\d+)/stream(?:/\d+)+'
```